src/forms.py

```diff
@@ -20,35 +20,34 @@ def validate(self) -> bool:
 
 class LoginForm(FlaskForm):
     """Login form"""
-    username = StringField('Użytkownik', validators=[DataRequired()])
+    name = StringField('Użytkownik', validators=[DataRequired()])
     password = PasswordField('Hasło', validators=[DataRequired()])
     remember_me = BooleanField('Pozostań zalogowany')
     submit = SubmitField('Zaloguj')
 
 
 class SignInForm(FlaskForm):
-    user = StringField(
+    name = StringField(
         'Użytkownik', validators=[
             DataRequired(), Length(3, 80),
             Regexp('^[A-Za-z0-9_]{3,}$',
-                   message='Dopuszczalne litery, '
-                           'cyfry oraz podkreślenia.')
+                   message='Dopuszczalne litery, cyfry oraz podkreślenia')
         ]
     )
     email = StringField('Email', validators=[DataRequired(), Length(1, 120),
                                              Email()])
     password = PasswordField(
         'Hasło', validators=[
             DataRequired(),
-            EqualTo('password2', message='Hasła muszą się zgadzać.')
```

Comment:
Jak się zgadzać? To było moje pytanie które powstało w mojej głowie podczas tej nieistotnej zmiany.

Reply:
no spoko :1st_place_medal: